### DISCUSSION TITLE
Is there any example project for flowbite-react?

### DISCUSSION THREAD

**Discussion by dimaslanjaka** [themesberg/flowbite-react#1070] [Q&A] [answered]
I was tried building layout from stratch with `flowbite-react` with follow guide from the `flowbite`, `flowbite-react`, `tailwind` docs. I never took it responsive across multiple platform (responsive viewer dev tools).

i often took these error:
- the image bigger than outer divs
- the body height not full
- the inner div overlaped height or width of outer div
- theme switcher switching background color only, keep text color in black which caused unreadable anymore
- flex class caused overlaped outer div also

is there any sample project ? or theme starter or anything.

I was seek on github for `flowbite-react`, but not all working with different `flowbite` and `flowbite-react` version. 

hmm i was confused

**Comment by SutuSebastian** [comment, marked as answer]
Sure we do, check out the [quickstart](https://flowbite-react.com/docs/getting-started/quickstart) page, which highlights the usage of [`create-flowbite-react`](https://www.npmjs.com/package/create-flowbite-react) CLI.

The projects are completely setup and ready to go with the latest version of `flowbite-react`.
